version.txt

```diff
@@ -1 +1 @@
-1.1.0
```

Comment:
@NNcrawler Can we purge and then add this to clean up version history?

Reply:
Okay, will purge this